client/sites/settings/v2/style.scss

```diff
@@ -24,6 +24,14 @@
 		min-height: 0;
 	}
 
+	// BaseControl currently has an override for the label to accommodate the use case where the label is shown next to the control.
+	// In site settings forms, the label is above the control for regular fields so we need to override this override for UI consistency.
+	.dataforms-layouts-regular__field .components-base-control__label {
+		font-size: 11px;
+		font-weight: 500;
+		text-transform: uppercase;
```

Comment:
I don't think it works on Reset and Delete site modals? The DOM is NOT inside `.dashboard-backport-site-settings-root`; it's in an overlay (portal).

Reply:
<img width="489" alt="image" src="https://github.com/user-attachments/assets/c7091dc2-1c99-46dc-a015-b06dd2b4bc54" />